src/Shared/Model/Mutators/AfterSeparatorMutator.cs

```diff
@@ -0,0 +1,34 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// --------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.CST.OpenSource.Model.Mutators
+{
+    /// <summary>
+    /// Generates mutations for if a character was changed after a separator was used.
+    /// NOTE: Doesn't currently support extra separators like the <see cref="SeparatorMutator"/> does.
+    /// </summary>
+    public class AfterSeparatorMutator : BaseMutator
+    {
+        public new string Mutator = "AFTER_SEPARATOR_CHANGE";
+
+        public override IEnumerable<(string Name, string Reason)> Generate(string arg)
+        {
+            foreach (var s in SeparatorMutator.separators)
+            {
+                var splits = arg.Split(s, StringSplitOptions.RemoveEmptyEntries);
+
+                if (splits.Length == 2)
+                {
+                    for (var c = 'a'; c <= 'z'; c++)
```

Comment:
Any reason we don't want to also iterate over uppercase letters or numbers?

Reply:
Most (all?) package managers don't care about casing.

Numbers is a good idea.